FeedSX/screens/createPost/index.tsx

```diff
@@ -0,0 +1,388 @@
+                    videoUrl={`${formattedMediaAttachments[0]?.attachmentMeta.url}`}
+                    showCancel
+                    onCancel={() => removeSingleAttachment()}
+                  />
+                )}
+              </>
+            )
+          ) : null}
+          {/* selected document view section */}
+          {formattedDocumentAttachments &&
+            formattedDocumentAttachments.length >= 1 && (
+              <LMDocument
+                attachments={formattedDocumentAttachments}
+                showCancel
+                onCancel={index => removeDocAttachment(index)}
+              />
+            )}
+          {/* added link preview section */}
+          {formattedMediaAttachments.length <= 0 &&
+            formattedDocumentAttachments.length <= 0 &&
+            linkPreviewData &&
+            linkPreviewData.length >= 1 && (
+              <LMLinkPreview
+                attachments={linkPreviewData}
+                showCancel
+                onCancel={() => {
+                  setLinkPreviewData([]);
+                }}
+              />
+            )}
+        </View>
+        {/* add more media button section */}
+        {(formattedMediaAttachments.length > 0 ||
+          formattedDocumentAttachments?.length > 0) && (
+          <LMButton
+            onTap={
+              formattedMediaAttachments.length > 0
+                ? () => handleGallery(SELECT_BOTH)
+                : formattedDocumentAttachments.length > 0
+                ? () => handleDoc()
+                : () => {}
+            }
+            icon={{
+              assetPath: require('../../assets/images/plusAdd_icon3x.png'),
+              type: 'png',
+              height: 20,
+              width: 20,
+            }}
+            text={{
+              text: ADD_MORE_MEDIA,
+              textStyle: {
+                fontSize: 14,
+                fontWeight: '500',
+                color: '#5046E5',
+                marginLeft: 5,
+              },
+            }}
+            buttonStyle={styles.addMoreButtonView}
+          />
+        )}
+      </ScrollView>
+      {/* selection options section */}
+      {showOptions && (
+        <View style={styles.selectionOptionsView}>
+          {/* add photos button */}
+          <TouchableOpacity
+            style={styles.optionItemView}
+            onPress={() => {
+              handleGallery(SELECT_IMAGE);
+            }}>
+            <LMIcon
+              type="png"
+              assetPath={require('../../assets/images/gallery_icon3x.png')}></LMIcon>
+            <LMText
+              text={ADD_IMAGES}
+              textStyle={{marginLeft: 8, color: '#222020'}}
+            />
+          </TouchableOpacity>
+          {/* add video button */}
+          <TouchableOpacity
+            style={styles.optionItemView}
+            onPress={() => {
+              handleGallery(SELECT_VIDEO);
+            }}>
+            <LMIcon
+              type="png"
```

Comment:
import from string.ts file

Reply:
this is default value already set in UI kit , so we are always getting it in auto suggestion.